packages/blazor-workspace/Tests/NimbleBlazor.Tests.Acceptance.Shared/MainLayout.razor

```diff
@@ -1,4 +1,5 @@
 ﻿@using Microsoft.AspNetCore.Components
+@using Microsoft.AspNetCore.Components.Web
```

Comment:
Not sure why this wasn't needed before. I was getting various build errors and squiggles without it.